.eslintrc.js

```diff
@@ -44,6 +44,10 @@ module.exports = {
     'cypress/assertion-before-screenshot': 'warn',
     'cypress/no-force': 'warn',
     'cypress/no-async-tests': 'error',
+    // default no-unused-vars doesn't handle when vars are only used in types, use extended version
+    // See: https://typescript-eslint.io/rules/no-unused-vars
+    'no-unused-vars': 'off',
+    '@typescript-eslint/no-unused-vars': 'error'
```

Comment:
@mengweieric Can you please review this change in es lint?

Reply:
also add this to ignore `_*` variables that should be unused? https://github.com/joshuali925/observability-langchain/blob/ebdd5152a0d5cdeddea56f2e93ecc882b045833e/.eslintrc.js#L27-L31
```js
    '@typescript-eslint/no-unused-vars': [
      'error',
      {
        argsIgnorePattern: '^_',
        varsIgnorePattern: '^_',
        caughtErrorsIgnorePattern: '^_',
      },
    ],
```

i remember eric had a PR for this

Reply:
@Swiddis Can you please undo the lint changes and rebase from the latest 2.x?

Reply:
Applied the changes. I notice that in the docs from the comment they explicitly disable the old unused-vars lint, but we keep both enabled. I wonder if that difference matters? The config from Eric's PR at least seems to not report errors on the file I was fixing here so it probably isn't important for now.

Reply:
it's contained by recommended